meson.build

```
@@ -37,14 +37,16 @@ endforeach
 
 first_run_template = files(get_option('first_run_template'))
 sdk_update_template = files(get_option('sdk_update_template'))
+first_run_filename = '.'.join(['first_run', get_option('first_run_filename_extension')])
+sdk_update_filename = '.'.join(['sdk_update', get_option('sdk_update_filename_extension')])
```

You can use `fs` module to get the file basename, then it would be possible to keep the original filename.
If you add `fs = import('fs')` somewhere at the top of the `meson.build`, this should work:
```suggestion
first_run_filename = fs.name(get_option('first_run_template'))
sdk_update_filename = fs.name(get_option('sdk_update_template'))
```

---

Yes, I think your suggestion is more clean =)